lib/modules/story_editor/widgets/actor_picker_sheet.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:andax/models/actor.dart';
+import 'package:andax/modules/story_editor/screens/story_editor.dart';
+import 'package:andax/modules/story_editor/widgets/actor_tile.dart';
+import 'package:andax/shared/widgets/modal_picker.dart';
+import 'package:andax/shared/widgets/rounded_back_button.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'actor_editor_dialog.dart';
+
+void showActorPickerSheet(
+  BuildContext context,
+  ValueSetter<Actor?> onSelect, [
```

Comment:
Same thing as before about returning a Future instead of accepting `ValueSetter`